### PR TITLE
DOC-370 | ArangoGraph - certificates info and encryption

### PR DESCRIPTION
**Pull request by nerpaula**

### CODE REVIEW

File: arangograph/projects.md [hunk 1]
```diff
@@ -115,9 +121,14 @@ certificate generation, both the default and high port serve the same
 self-signed certificate.
 {% endhint %}
 
-When a private endpoint is being used, you can specify alternate DNS names,
-which are added to the self-signed certificate only as Subject Alternative
-Name (SAN).
+When using private endpoints, notebooks, and metrics, you can specify
```
Comment (Robert-Stam) [inline arangograph/projects.md:124]: can you add a note in the PE section, so a customer can find this info more easily?

**Review by Robert-Stam — APPROVED** [review]
See remark, further LGTM

File: arangograph/projects.md (hunk 1 above)
Comment (nerpaula) [reply arangograph/projects.md:124]: done

File: 3.10/security-encryption.md [hunk 2]
```diff
@@ -28,6 +28,11 @@ performance and resistance to side-channel attacks.
 
 The encryption feature is supported by all ArangoDB deployment modes.
 
+{% hint 'info' %}
+The ArangoGraph Insights Platform has encryption set on by default and cannot
```
Comment (Simran-B) [inline 3.10/security-encryption.md:32]: Maybe mention that this refers to encryption at rest (encrypted on disk) as well as in transit (encrypted communication)?

Comment (nerpaula) [reply 3.10/security-encryption.md]: yes, added it.

**Pull request merged**